### PR TITLE
Remove ellipses from links and buttons

### LINKED ISSUE #2332: Remove ellipses
screen shot 2015-05-18 at 1 43 09 pm

Remove ellipses from link (and all links/buttons within the UI).

### PR DESCRIPTION
**Pull request by spadgett**
Fixes #2332

Projects page:

openshift_web_console

Browse builds:

openshift_web_console

Browse image streams:

openshift_web_console

Select template:

openshift_web_console

### CODE REVIEW

**Comment by spadgett** [comment]
I haven't changed the `Create...` button in the project nav because we haven't decided on a label.

File: assets/app/views/_tasks.html [hunk 1]
```diff
@@ -20,8 +20,8 @@
           </div>
           <div ng-show="task.hasErrors">
             <a href="javascript:;" ng-click="toggleExpand()">
-              <span ng-hide="expanded">More details...</span>
-              <span ng-show="expanded">Less details...</span>
+              <span ng-hide="expanded">More details</span>
+              <span ng-show="expanded">Less details</span>
```
Comment (liggitt) [inline assets/app/views/_tasks.html]: I think the link should be a verb... "Show details"/"Hide details"

**Comment by spadgett** [comment]
Updated to "Show details" and "Hide details"

**Comment by liggitt** [comment]
LGTM, this ready?

**Comment by spadgett** [comment]
@liggitt ready

**Comment by liggitt** [comment]
[merge]

**Comment by spadgett** [comment]
```
--- FAIL: TestRunBuild (0.75s)
	newapp_test.go:835: successful ruby app generation: Error mismatch! Expected <nil>, got no image or template matched "openshift/mongodb-24-centos7": can't connect to "": error retrieving tag: server returned 401
W0708 12:50:34.092295   12307 newapp.go:510] Could not find an image match for "mysql". Make sure that a Docker image with that tag is available on the OpenShift node for the build to succeed.
FAIL
coverage: 70.4% of statements
FAIL	github.com/openshift/origin/pkg/generate/app/cmd	4.080s
```

**Pull request merged**